package.json

```diff
@@ -63,9 +67,12 @@
     "validate-commit-message": "^3.0.1"
   },
   "peerDependencies": {
+    "connected-react-router": "^6.9.1",
```

Comment:
```
Error: src/modules/location/sagas.ts(6,8): error TS2307: Cannot find module 'connected-react-router' or its corresponding type declarations.
Error: src/modules/location/sagas.ts(15,53): error TS7057: 'yield' expression implicitly results in an 'any' type because its containing generator lacks a return-type annotation.
Error: Process completed with exit code 2.
```

I believe this error is because peer deps have to be added as dev dependencies as well or they will not be installed on the project when working locally